core/src/main/resources/hudson/model/Node/help-numExecutors.html

```diff
@@ -9,10 +9,10 @@
   the second build could take advantage of the spare I/O capacity at that
   moment.
   <p>
-  Agents must have at least one executor. To temporarily prevent any builds from
-  being executed on an agent, use the <i>Mark this node temporarily offline</i>
-  button on the agent's page.
+  Agents (nodes that are not the master) must have at least one executor.
+  To temporarily prevent any builds from being executed on an agent, use the
+  <i>Mark this node temporarily offline</i> button on the agent's page.
   <p>
-  This does not apply to the Jenkins master &mdash; setting the number of
-  executors to zero will prevent any builds from being executed on the master.
+  For the master, set the number of executors to zero to prevent it from
```

Comment:
I would explicitly mention that the master will be still able to run flyweight tasks like Pipeline's top-level task

Reply:
Thanks. Will do. Since I've hit those edges.